### DISCUSSION TITLE
Deep Links from Instagram in-app Browser aren't working

### DISCUSSION THREAD

**Discussion by Emtschikay** [JulianPrieber/littlelink-custom#769] [Issues]
Hello,

I recently switched from Linktree to Instagram. Whenever a user clicked on the link in my bio, they were presented with the Linktree page in the in-app Browser of Instagram, and when clicking on e.g. Spotify, they app was opened. 
Now, with Linkstack, this doesn't work anymore. When clicking on the Spotify link in the Instagram browser, it redirects to the Spotify webplayer still inside the same browser window. 

Any tips on solving this are highly appreciated.

Thank you very much!

**Comment by simonives** [comment]
I'm unsure on this one - I hadn't even tested this before on my end.

I've just opened the Instagram app on my iPhone, clicked the link to my LinkStack site, and clicked a few links where I have the corresponding app installed. I don't use Spotify, so I couldn't test that, however links for Signal, WhatsApp, and Telegram all opened the app on my device rather than navigating to the corresponding websites.

If this is a user reported issue then it may have more to do with how the user has configured link handling on their own device ([e.g. here](https://www.lambdatest.com/software-testing-questions/how-to-open-link-in-app-instead-of-browser)).
